front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/ParticipantsWidget/index.tsx

```diff
@@ -1,15 +1,35 @@
 import React from 'react';
 
+import { Box } from '@citizenlab/cl2-component-library';
+
+import usePhases from 'api/phases/usePhases';
+
+import { hasEmbeddedSurvey } from 'containers/ProjectsShowPage/shared/header/utils';
+
+import Warning from 'components/UI/Warning';
+
+import { useIntl } from 'utils/cl-intl';
+
 import Card from '../../_shared/Card';
 import messages from '../messages';
 
 import ChartWidgetSettings from './ChartWidgetSettings';
 import ParticipantsCard from './ParticipantsCard';
 import { Props } from './typings';
 
-const ParticipantsWidget = ({ title, ...props }: Props) => {
+const ParticipantsWidget = ({ title, projectId, ...props }: Props) => {
+  const { formatMessage } = useIntl();
+  const { data: phases } = usePhases(projectId);
+
   return (
     <Card title={title} pagebreak>
+      {hasEmbeddedSurvey(phases?.data) && (
+        <Box mb="16px">
+          <Warning>
+            {formatMessage(messages.adminInaccurateParticipantsWarning)}
+          </Warning>
+        </Box>
```

Comment:
If this report is shown in the front office to residents, will this warning message also show there? Or is it only shown in the back office? :)

Reply:
Yeah I'm a bit conflicted as well :thinking: If I remember correctly, we've seen some really keen residents in the past who actually calculated participation numbers and saw discrepancies with reported numbers and called out our clients. In which case showing this warning to residents might be best?

I'm not sure though.. maybe Rob or Irene would have an opinion here?